bndtools.api/src/org/bndtools/api/ProjectPaths.java

```diff
@@ -75,16 +91,26 @@ public String getTitle() {
         return title;
     }
 
+    @Deprecated
     public String getSrc() {
-        return src;
+        return srcs.get(0);
+    }
+
+    public List<String> getSrcs() {
+        return srcs;
     }
 
     public String getBin() {
         return bin;
     }
 
+    @Deprecated
     public String getTestSrc() {
-        return testSrc;
+        return testSrcs.get(0);
```

Comment:
You require `srcs` to be non-empty, but you don't require `testSrcs` to be non-empty. So this line can throw an exception.